packages/patternfly-4/react-core/scripts/convertComponentHelpers.js

```diff
@@ -1,98 +1,87 @@
 function getMatches(string, pattern) {
```

Comment:
Seems like the changes to convertComponentHelpers.js and convertComponentToTS.js would be better served as part of an independent PR.

Reply:
Reverted.